packages/editor/src/models/tree/SpecTreeNode.ts

```diff
@@ -0,0 +1,11 @@
+import { SpecTreeNodeParam } from './SpecTreeNodeParam';
+import { HttpMethod } from '../HttpMethod';
+
+export interface SpecTreeNode {
+  readonly path: string;
+  readonly name?: string;
+  readonly method?: HttpMethod;
+  readonly jsonPointer: string;
+  readonly children?: ReadonlyArray<SpecTreeNode>;
+  readonly parameters?: ReadonlyArray<SpecTreeNodeParam>;
```

Comment:
Misconfigured `eslint` rules) Optional fields must be placed last

```suggestion
  readonly jsonPointer: string;
  readonly name?: string;
  readonly method?: HttpMethod;
  readonly children?: ReadonlyArray<SpecTreeNode>;
  readonly parameters?: ReadonlyArray<SpecTreeNodeParam>;
```